pallets/ddc-verification/src/lib.rs

```diff
@@ -1268,6 +1348,14 @@ pub mod pallet {
 				Percent::from_percent(T::MAJORITY),
 			)?;
 
+			let _minimum_sub_trees = Self::fetch_minimum_sub_aggregates(
+				cluster_id,
+				era_activity.id,
+				customers_activity_in_consensus.clone(),
```

Comment:
so, we are working with sub aggregates only for buckets that are already in consensus? i'm not sure it's what we want.

what the reason behind it?